app/models/measure_type.rb

```diff
@@ -6,7 +6,7 @@ class MeasureType < Sequel::Model
   IMPORT_MOVEMENT_CODES = [0, 2].freeze
   EXPORT_MOVEMENT_CODES = [1, 2].freeze
   THIRD_COUNTRY = %w[103 105].freeze # 105 measure types are for end use Third Country duties. 103 are for everything else
-  VAT_TYPES = %w[VTA VTE VTS VTZ 305].freeze
+  VAT_TYPES = %w[305].freeze
```

Comment:
The measure_type_id would never have been VTA, VTE, VTS or VTZ from my observations of the data.

Reply:
Ok, check with Matt just in case.

Reply:
XI:

```sql
SELECT COUNT(1) FROM measures WHERE measure_type_id IN ('VTA', 'VTE', 'VTS', 'VTZ');
 count
--------
 386433
(1 row)
```

UK
```sql
SELECT COUNT(1) FROM measures WHERE measure_type_id IN ('VTA', 'VTE', 'VTS', 'VTZ');
 count
-------
     0
(1 row)
```